Marked 0.12, 0.13, 0.14, and 0.15 as unsupported and 0.19 under development (#1390)
keptn/keptn.github.io: content/docs/0.19.x/_index.md

sidebar_multicard: true
icon: docs
hide: true

> develop: This documentation is currently under development and officially released with the next Keptn release. 

Reviewer: ```suggestion
develop: This documentation is currently under development for a future Keptn release. 
```
We can't guarantee that this will be the next release.  We have hide: true set so customers shouldn't see this anyhow.

Author: Fixed it. Thx!
